src/EnergyPlus/ChilledCeilingPanelSimple.hh

```diff
@@ -0,0 +1,347 @@
+			Real64 const FracConvect,
+			Real64 const FracDistribPerson,
+			Array1< Real64 > const & FracDistribToSurf,
+			Real64 const TotPower,
+			Real64 const Power,
+			Real64 const ConvPower,
+			Real64 const RadPower,
+			Real64 const TotEnergy,
+			Real64 const Energy,
+			Real64 const ConvEnergy,
+			Real64 const RadEnergy,
+			int const LoopNum, // plant loop index
+			int const LoopSideNum, // plant loop side index
+			int const BranchNum, // plant loop branch index
+			int const CompNum, // plant loop component index
+			int const CoolingPanelLoadReSimIndex,
+			int const CoolingPanelMassFlowReSimIndex,
+			int const CoolingPanelInletTempFlowReSimIndex
+		) :
+			EquipID( EquipID ),
+			EquipType( EquipType ),
+			Schedule( Schedule ),
+			SurfaceName( SurfaceName ),
+			SurfacePtr( SurfacePtr ),
+			ZonePtr( ZonePtr ),
+			SchedPtr( SchedPtr ),
+			WaterInletNode( WaterInletNode ),
+			WaterOutletNode( WaterOutletNode ),
+			TotSurfToDistrib( TotSurfToDistrib ),
+			ControlCompTypeNum( ControlCompTypeNum ),
+			CompErrIndex( CompErrIndex ),
+			ControlType( ControlType ),
+			ColdSetptSched( ColdSetptSched ),
+			ColdSetptSchedPtr( ColdSetptSchedPtr ),
+			CondCtrlType( CondCtrlType ),
+			CondDewPtDeltaT( CondDewPtDeltaT ),
+			CondErrIndex( CondErrIndex ),
+			ColdThrottlRange( ColdThrottlRange ),
+			RatedWaterTemp( RatedWaterTemp ),
+			RatedCapacity( RatedCapacity ),
+			UA( UA ),
+			Offset( Offset ),
+			WaterMassFlowRate( WaterMassFlowRate ),
+			WaterMassFlowRateMax( WaterMassFlowRateMax ),
+			RatedWaterFlowRate( RatedWaterFlowRate ),
+			WaterVolFlowRateMax( WaterVolFlowRateMax ),
+			WaterInletTempStd( WaterInletTempStd ),
+			WaterInletTemp( WaterInletTemp ),
+			WaterInletEnthalpy( WaterInletEnthalpy ),
+			WaterOutletTempStd( WaterOutletTempStd ),
+			WaterOutletTemp( WaterOutletTemp ),
+			WaterOutletEnthalpy( WaterOutletEnthalpy ),
+			RatedZoneAirTemp( RatedZoneAirTemp ),
+			FracRadiant( FracRadiant ),
+			FracConvect( FracConvect ),
+			FracDistribPerson( FracDistribPerson ),
+			FracDistribToSurf( FracDistribToSurf ),
+			TotPower( TotPower ),
+			Power( Power ),
+			ConvPower( ConvPower ),
+			RadPower( RadPower ),
+			TotEnergy( TotEnergy ),
+			Energy( Energy ),
+			ConvEnergy( ConvEnergy ),
+			RadEnergy( RadEnergy ),
+			LoopNum( LoopNum ),
+			LoopSideNum( LoopSideNum ),
+			BranchNum( BranchNum ),
+			CompNum( CompNum ),
+			CoolingPanelLoadReSimIndex( CoolingPanelLoadReSimIndex ),
+			CoolingPanelMassFlowReSimIndex( CoolingPanelMassFlowReSimIndex ),
+			CoolingPanelInletTempFlowReSimIndex( CoolingPanelInletTempFlowReSimIndex )
+		{}
+
+	};
+
+	// Object Data
+	extern Array1D< CoolingPanelParams > CoolingPanel;
+
+	// Functions
+
+	void
+	SimCoolingPanel(
+		std::string const & EquipName,
+		int const ActualZoneNum,
+		int const ControlledZoneNum,
+		bool const FirstHVACIteration,
+		Real64 & PowerMet,
+		int & CompIndex
+	);
+
+	void
+	GetCoolingPanelInput();
+
+	void
+	InitCoolingPanel(
```

Comment:
@Myoldmopar Yes, I would appreciate help on working that up as well as understanding what that means.  I kinda of dislike feeling like I am an old dog trying to learn new tricks, but I DO want to figure this out so that I can continue to be a productive team member and don't mess a bunch of stuff up.